### ISSUE TITLE
Radarr downloads better quality but doesn't replace old file

### ISSUE THREAD

**Issue by JohanPotgieter** [Radarr/Radarr#1604]
**Description:**
Radarr downloaded a new file to replace the old SD version. It moved it (and renamed it) after downloaded completed but it still shows the old file. It also now says the file is missing. Radarr also deleted the old file as it's missing from the disk now.
![missing file](https://cloud.githubusercontent.com/assets/12456610/26679001/9aeedfd2-4716-11e7-99a4-fc6a289785bd.JPG)

The download finished about 6 hours ago and still no change. Looking at the image above the file is shown as missing. The other odd thing is, when I do a search the red dot says "existing file meets cutoff...". This to me means that Radarr is aware of the new file. See image below.

![meets cutoff](https://cloud.githubusercontent.com/assets/12456610/26679200/683237fa-4717-11e7-9052-877f131eb53b.JPG)

**Radarr Version:**
0.2.0.696

**Logs:**
```
NzbSearchService            Searching 1 indexers for [Star Trek Beyond]	14:17
MovieSearchService	Movie search completed. 1 reports downloaded.	14:17
DownloadService	        Report sent to Sabnzbd. Star.Trek.Beyond.2016.720p.BluRay.DD-EX.5.1.x264-NCmt	14:17
Sabnzbd	                        Adding report [Star.Trek.Beyond.2016.720p.BluRay.DD-EX.5.1.x264-NCmt] to the queue.	14:17
DownloadDecisionMaker	Processing 99 releases
```

Please use the search bar and make sure you are not submitting an already submitted issue.
Visit our [Discord server](https://discord.gg/NWYch8M) for support or longer discussions.

**Comment by JohanPotgieter** [comment]
Believe it or not but a few minutes after I posted this the item was updated. I did a search for the movie and when it come up it was updated. See Image below.

![mysterious update](https://cloud.githubusercontent.com/assets/12456610/26679881/518fa318-471a-11e7-8bab-3e2ce5d318de.JPG)

I do think that someone still needs to look into this as Radarr knew about the new file after download so some kind of update should have been done automatically.
